omnisafe/algorithms/off_policy/ddpg.py

```diff
@@ -313,16 +313,42 @@ def learn(self):
                 self.log(epoch, steps)
         return self.actor_critic
 
-    def update(self, data):
+    def update(self, data: dict) -> None:
         """Update.
+        Update step contains three parts:
+
+        #.  Update value net by :func:`update_value_net()`
+        #.  Update cost net by :func:`update_cost_net()`
+        #.  Update policy net by :func:`update_policy_net()`
+        #.  Update target net by :func:`polyak_update_target()`
```

Comment:
1. A extra line before the description.
2. Use `:meth:` without brackets.

```suggestion

        Update step contains three parts:

        #.  Update value net by :meth:`update_value_net`
        #.  Update cost net by :meth:`update_cost_net`
        #.  Update policy net by :meth:`update_policy_net`
        #.  Update target net by :meth:`polyak_update_target`
```